src/components/summary/examples/hub/index.njk

```diff
@@ -6,6 +6,7 @@
             {
                 "groups": [
                     {
+                    "headers":["Name of section or person", "Section progress", "Access section"],
```

Comment:
Can runner provide just values and not keys? Maybe something to check.

Reply:
Should be able to I would think but ill check with the runner guys

Reply:
Spoken to the runner guys and they're happy with this implementation